lld/COFF/SymbolTable.cpp

```diff
@@ -502,6 +503,11 @@ void SymbolTable::resolveRemainingUndefines() {
     // This odd rule is for compatibility with MSVC linker.
     if (name.starts_with("__imp_")) {
       Symbol *imp = find(name.substr(strlen("__imp_")));
```

Comment:
Side note, I find the variable name `imp` here kinda misleading, as it's the imp-less symbol! (We can consider changing it in a separate patch later.)